portfolio/src/main/java/com/google/sps/data/comments.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto2";
+
+package tutorial;
+
+option java_package = "com.google.sps.data";
+option java_outer_classname = "TotalCommentProtos";
+
+message Comment {
+  optional string name = 1;
+  required string data = 2;
```

Comment:
The Google style guide discourages using 'required' fields. It seems counterintuitive but it has to do with long term maintainability of code. This is only FYI, you don't have to change anything here for this pull request.

Reply:
Thank you! I will keep that in mind.